pkg/ride/generate/main.go

```diff
@@ -361,6 +363,7 @@ func evaluationCatalogueV3EvaluatorV1() map[string]int {
 	m["TransferSet"] = 0
 	m["ScriptTransfer"] = 0
 	m["ScriptResult"] = 0
+	// TODO add evaluation for Transfer constructor
```

Comment:
Evaluation complexity of all constructors is 0, but you have to add it explicitly otherwise it will be equal to estimation complexity.